packages/grafana-ui/src/components/Logs/LogRows.tsx

```diff
@@ -139,6 +146,7 @@ class UnThemedLogRows extends PureComponent<Props, State> {
                   onClickFilterLabel={onClickFilterLabel}
                   onClickFilterOutLabel={onClickFilterOutLabel}
                   getFieldLinks={getFieldLinks}
+                  logsSortOrder={logsSortOrder}
                 />
```

Comment:
Passing `logsSortOrder` to `LogRow` so it can be passed on `LogsRowContextProvider`.